### PR TITLE
Update to the new ngraph batch norm API

### PR DESCRIPTION
**Pull request by csullivan**

### CODE REVIEW

**Comment by avijit-nervana** [comment]
@csullivan The formatting check failed. Please run the maint/apply-format-check.sh that will fix these.

**Comment by sayantan-nervana** [comment]
It is failing still with batchnorm errors. The logs are here: https://aipg-rancher.intel.com/jenkins/algo/job/ngraph-tf-unittest/1024/artifact/publish/logs/build-ngraph-tf-py2-log.txt

Some error prints:
```
/home/dockuser/ngraph-tf/src/ngraph_builder.cc: In function 'tensorflow::Status tensorflow::ngraph_bridge::TranslateFusedBatchNormOp(const tensorflow::Node*, const std::vector<const tensorflow::Tensor*>&, tensorflow::ngraph_bridge::Builder::OpMap&)':
/home/dockuser/ngraph-tf/src/ngraph_builder.cc:1723:33: error: 'BatchNormTraining' is not a member of 'ngraph::op'
     ng_batch_norm = make_shared<ng::op::BatchNormTraining>(tf_epsilon, ng_scale,
                                 ^
/home/dockuser/ngraph-tf/src/ngraph_builder.cc:1723:33: error: 'BatchNormTraining' is not a member of 'ngraph::op'
/home/dockuser/ngraph-tf/src/ngraph_builder.cc:1724:79: error: no matching function for call to 'make_shared(float&, std::shared_ptr<ngraph::Node>&, std::shared_ptr<ngraph::Node>&, std::shared_ptr<ngraph::Node>&)'
                                                            ng_offset, ng_input);
                                                                               ^
In file included from /usr/include/c++/5/memory:82:0,
                 from /home/dockuser/ngraph-tf/BUILD-BRIDGE/ngraph/ngraph_dist/include/ngraph/node.hpp:22,
                 from /home/dockuser/ngraph-tf/BUILD-BRIDGE/ngraph/ngraph_dist/include/ngraph/builder/autobroadcast.hpp:20,
                 from /home/dockuser/ngraph-tf/BUILD-BRIDGE/ngraph/ngraph_dist/include/ngraph/ngraph.hpp:46,
                 from /home/dockuser/ngraph-tf/src/ngraph_builder.h:22,
                 from /home/dockuser/ngraph-tf/src/ngraph_builder.cc:17:
/usr/include/c++/5/bits/shared_ptr.h:632:5: note: candidate: template<class _Tp, class ... _Args> std::shared_ptr<_Tp1> std::make_shared(_Args&& ...)
     make_shared(_Args&&... __args)
     ^
/usr/include/c++/5/bits/shared_ptr.h:632:5: note:   template argument deduction/substitution failed:
/home/dockuser/ngraph-tf/src/ngraph_builder.cc:1724:79: error: template argument 1 is invalid
                                                            ng_offset, ng_input);
                                                                               ^
/home/dockuser/ngraph-tf/src/ngraph_builder.cc:1744:33: error: 'BatchNormInference' is not a member of 'ngraph::op'
     ng_batch_norm = make_shared<ng::op::BatchNormInference>(
                                 ^
/home/dockuser/ngraph-tf/src/ngraph_builder.cc:1744:33: error: 'BatchNormInference' is not a member of 'ngraph::op'
/home/dockuser/ngraph-tf/src/ngraph_builder.cc:1745:72: error: no matching function for call to 'make_shared(float&, std::shared_ptr<ngraph::Node>&, std::shared_ptr<ngraph::Node>&, std::shared_ptr<ngraph::Node>&, std::shared_ptr<ngraph::Node>&, std::shared_ptr<ngraph::Node>&)'
         tf_epsilon, ng_scale, ng_offset, ng_input, ng_mean, ng_variance);
                                                                        ^
In file included from /usr/include/c++/5/memory:82:0,
                 from /home/dockuser/ngraph-tf/BUILD-BRIDGE/ngraph/ngraph_dist/include/ngraph/node.hpp:22,
                 from /home/dockuser/ngraph-tf/BUILD-BRIDGE/ngraph/ngraph_dist/include/ngraph/builder/autobroadcast.hpp:20,
                 from /home/dockuser/ngraph-tf/BUILD-BRIDGE/ngraph/ngraph_dist/include/ngraph/ngraph.hpp:46,
                 from /home/dockuser/ngraph-tf/src/ngraph_builder.h:22,
                 from /home/dockuser/ngraph-tf/src/ngraph_builder.cc:17:
/usr/include/c++/5/bits/shared_ptr.h:632:5: note: candidate: template<class _Tp, class ... _Args> std::shared_ptr<_Tp1> std::make_shared(_Args&& ...)
     make_shared(_Args&&... __args)
     ^
/usr/include/c++/5/bits/shared_ptr.h:632:5: note:   template argument deduction/substitution failed:
/home/dockuser/ngraph-tf/src/ngraph_builder.cc:1745:72: error: template argument 1 is invalid
         tf_epsilon, ng_scale, ng_offset, ng_input, ng_mean, ng_variance);
                                                                        ^
/home/dockuser/ngraph-tf/src/ngraph_builder.cc: In function 'tensorflow::Status tensorflow::ngraph_bridge::TranslateFusedBatchNormGradOp(const tensorflow::Node*, const std::vector<const tensorflow::Tensor*>&, tensorflow::ngraph_bridge::Builder::OpMap&)':
/home/dockuser/ngraph-tf/src/ngraph_builder.cc:1812:40: error: 'BatchNormTrainingBackprop' is not a member of 'ngraph::op'
   ng_batch_norm_backprop = make_shared<ng::op::BatchNormTrainingBackprop>(
                                        ^
/home/dockuser/ngraph-tf/src/ngraph_builder.cc:1812:40: error: 'BatchNormTrainingBackprop' is not a member of 'ngraph::op'
/home/dockuser/ngraph-tf/src/ngraph_builder.cc:1813:78: error: no matching function for call to 'make_shared(float&, std::shared_ptr<ngraph::Node>&, std::shared_ptr<ngraph::Node>&, std::shared_ptr<ngraph::Node>&, std::shared_ptr<ngraph::Node>&, std::shared_ptr<ngraph::Node>&, std::shared_ptr<ngraph::Node>&)'
       tf_epsilon, ng_scale, ng_beta, ng_input, ng_mean, ng_variance, ng_delta);
                                                                              ^
```
